cd/verify.sh

```diff
@@ -16,7 +16,7 @@ if [[ "$RELEASE" != 'true' ]] && [[ "$TRAVIS_PULL_REQUEST" != 'false' ]]; then
         source ~/.nvm/nvm.sh
 
         # Update nodejs to the latest version
-        nvm install node
+        nvm install 13.13.0
```

Comment:
It may be worth switching this to use the current LTS version as opposed to a specific version.

Reply:
```suggestion
        nvm install --lts
```

Reply:
After further consideration, it has been communicated that the lts version is incompatible with some items in the analytic ui project, as such we should perhaps keep the fixed version number for now. Alternatively, we could specify a conditional command specific to the analytics ui.

Reply:
Hi - I tried with `lts` and Travis seems to terminate with no explanation why in the logs (I think it might hang and then timeout). I've also tried `--lts` and Travis doesn't recognise it as an option.

I've also tried reverting back to the original `nvm install node` that installs version 14.4.0 . But the 'latest' Sass version in Analytics is only compatible with versions 13.x.x and not 14.x.x - so I stuck with the latest 13.x.x version 

It's a temporary fix in Analytics to ensure the pipeline still builds for this Pull Request